.github/workflows/scheduled.yml

```diff
@@ -42,8 +42,37 @@ jobs:
       - name: cargo update
         if: hashFiles('server/Cargo.lock') != ''
         run: cargo update
+      - name: Download DynamoDB Local
+        if: hashFiles('server/Cargo.lock') != ''
+        run: |
+          wget https://d1ni2b6xgvw0s0.cloudfront.net/v2.x/dynamodb_local_latest.tar.gz
+          wget https://d1ni2b6xgvw0s0.cloudfront.net/v2.x/dynamodb_local_latest.tar.gz.sha256
+          sha256sum --check dynamodb_local_latest.tar.gz.sha256
+          mkdir dynamodb-local
+          tar xfv dynamodb_local_latest.tar.gz --directory=dynamodb-local
+          ls -lah dynamodb-local
+      - name: Install Java SDK
+        if: hashFiles('server/Cargo.lock') != ''
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'oracle'
+          java-version: '21'
+      - name: Launch DynamoDB Local
+        if: hashFiles('server/Cargo.lock') != ''
+        run: |
+          cd dynamodb-local
+          java -Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar -sharedDb &
+      - name: Create tables and indexes in DynamoDb Local
+        if: hashFiles('server/Cargo.lock') != ''
+        run: ./run-migrations.sh http://localhost:8000
+        env:
+          AWS_ACCESS_KEY_ID: carpe
+          AWS_SECRET_ACCESS_KEY: diem
+          AWS_DEFAULT_REGION: dead-poets-society-eu-east-1
```

Comment:
Feels weird for this to be required here but then to use `local` for run. Can we make `run-migrations.sh` set these automatically to the right values?

Reply:
I decided not to, because `run-migrations.sh` has now got only one concern - create tables and indexes and does not assume anything about the environment. It is the other script (run-dynamodb-local) that sets all those env vars and calls the `run-migrations.sh` internally.  Setting those vars additionally on the CI did not look weird to me actually, because we are not making folks to do this on their workstations to be able to contribute to the project.